cypress/integration/immutableDatabaseTests/dropdown.ts

```diff
@@ -252,6 +252,74 @@ describe('Dropdown test', () => {
       // One pending org request should be visible
       cy.get('#my-pending-org-card-0').should('be.visible');
     });
+
+    it('Should be able to delete pending/reject org request', () => {
+      // Both pending and rejected orgs should be visible
+      cy.get('#my-pending-org-card-0').should('be.visible');
+      cy.get('#my-rejected-org-card-0').should('be.visible');
+
+      // New mocked memberships after deleting the rejected organization
+      let memberships = [
```

Comment:
It would be better if this were const instead of let. It's harder to follow code where variables are changing values.

Either create a different variable for the post-DELETE response, or filter this variable.

Reply:
Done, created a separate variable for the post-DELETE response.